byterun/ints.c

```diff
@@ -286,13 +286,13 @@ static int32_t caml_swap32(int32_t x)
 }
 
 value caml_int32_direct_bswap(value v)
-{ return caml_swap32(v); }
+{ return caml_swap32((int32_t) v); }
```

Comment:
What about the implicit cast on return? `caml_swap32` returns a `int32_t`, not a `value`.

Reply:
Indeed I only looked at 64->32 bit conversions -- those that lose information and are better made explicit.  int32_t to value doesn't lose information, so perhaps it can remain implicit.  I don't feel strongly about it either way.

Reply:
The asymmetry just feels slightly odd, but it can stay this way.